das/daser.go

```diff
@@ -17,25 +18,28 @@ import (
 
 var log = logging.Logger("das")
 
+const (
+	samplingRange = 1
```

Comment:
We should increase the value here. It feels like it should be close to the number of headers we retrieve in the batch via HeaderEx, but this is better to be analyzed.

Reply:
Increased to 64, further tweaking performance testing is needed to find optimal values